css/templates/travel/components/_featured.css

```diff
@@ -0,0 +1,145 @@
+/**
+  * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS-IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+.travel-featured-heading {
+  letter-spacing: none;
+}
+
+.travel-featured-grid {
+  margin: calc(2 / 16rem);
+}
+
+.travel-featured-tile {
+  margin: calc(2 / 16rem);
+}
+
+.travel-featured-tile-heading {
+  letter-spacing: 0.1em
+}
+
+.travel-featured-tile:hover,
+.travel-featured-tile:focus {
+  z-index: 1;
+}
+
+.travel-featured-tile amp-img {
+  transition: transform 500ms ease-out, box-shadow 500ms ease-out,
+}
+
+.travel-featured-tile img,
+.travel-featured-tile img {
+  transition: transform 2s ease-out;
+  transform-origin: 50% 66%;
+}
+
+.travel-featured-tile:hover amp-img,
+.travel-featured-tile:focus amp-img {
+  box-shadow: 0 0.6rem 2rem var(--box-shadow-color-dark);
+  transform: scale(1.05);
+}
+
+.travel-featured-tile:focus img,
+.travel-featured-tile:hover img {
+  transform: scale(1.3);
+}
+
+.travel-featured-tile .travel-featured-overlay {
+  color: white;
+  transition: transform 2s ease-out;
+}
+
+.travel-featured-tile:hover .travel-featured-overlay {
+  transform: translateY(1rem);
+}
+
+.travel-featured-tile amp-img::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding-bottom: 75%;
+  opacity: 0.9;
+  background-image: linear-gradient(to bottom, currentColor 0%, transparent 100%);
+}
+
+@media (--breakpoint-sm) {
+  .travel-featured-grid {
+    margin: calc(4 / 16rem);
+  }
+
+  .travel-featured-tile {
+    margin: calc(4 / 16rem);
+  }
+}
+
+@media (--breakpoint-md) {
+  .travel-featured::after {
+    content: '';
+    position: absolute;
+    width: 100%;
+    z-index: -1;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 112px;
+    margin-bottom: -8px;
+    background-color: var(--purple);
+  }
+
+  .travel-featured-grid {
+    margin: calc(6 / 16rem);
+  }
+
+  .travel-featured-tile {
+    margin: calc(6 / 16rem);
+  }
+}
+
+@media (--breakpoint-lg) {
+  .travel-featured-grid {
+    margin: calc(8 / 16rem);
+  }
+
+  .travel-featured-tile {
+    margin: calc(8 / 16rem);
+  }
+}
+
+.travel-featured-color-blue, .travel-featured-color-blue:visited {
```

Comment:
did we not declare similar stuff on other css files?, if we are reusing the same colors we need to use variables , else we are good here.

Reply:
These colors are unique